### PR TITLE
Webpage Set Up

### PR DESCRIPTION
**Pull request by piero-vic**
# Webpage Set Up
- Add README.md
- Add index.html
- Add css/style.css

### CODE REVIEW

**Review by piero-vic — COMMENTED** [review]
Hi,  👋

Good job so far. There are a couple of things that could be improved.

## To highlight
✔️ Good use of semantic tags.  
✔️ You followed best practices regarding how to organize your files.

## To change
See the comments for more details on things that require changes.
Good luck! 😉

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    <a href="www.google.com"></a>
+
+  </div>
+
+
+</section>
+
+<a id="profile-link" href="https://github.com/sj1978/anibal-mohamed19" target="_blank">GitHub</a>
+
```
[the remark this answers is not in the document]
Comment (vichuge) [reply index.html:32]: - [ ] Please, explain why this is important. 🐱

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    nav {
+      position: fixed;
+      width: 100%;
+      top: 0;
+      left: 0;
+      background-color: #cc0000;
+      color: white;
+      font-family: 'Exo 2', sans-serif;
+      padding: 1em;
+    }
```
[the remark this answers is not in the document]
Comment (vichuge) [reply css/style.css:20]: - [ ] Please, explain why this is important. 🐱

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (EricMbouwe) [reply css/style.css:20]: - Well done

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (EricMbouwe) [reply index.html:32]: - Well done! Everything is there 💪

File: css/style.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (EricMbouwe) [reply css/style.css:20]: - Good job making the changes. The comment is clear and you provided the why of your requested changes.

**Review by EricMbouwe — APPROVED** [review]
Hi @piero-vic, 

# Status: Approved✔️

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**